File: eshop-integration/eAPI/v1/java/eAPI-v1-connector/src/main/java/cz/monetplus/mips/eapi/v1/connector/entity/PayRes.java
package cz.monetplus.mips.eapi.v1.connector.entity;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
public class PayRes extends SignBase {

	private static final long serialVersionUID = -3825192932302805075L;
	
	@XmlElement
	public String payId;

	@XmlElement
	public int resultCode;

	@XmlElement
	public String resultMessage;

	@XmlElement
	public Integer paymentStatus;

	@XmlElement
	public String authCode;

	public PayRes() {
		
	}

	@Override
	public String toSign() {
		StringBuffer sb = new StringBuffer();
		add(sb, payId);
		add(sb, dttm);
		add(sb, resultCode);
		add(sb, resultMessage);
		add(sb, paymentStatus);
		add(sb, authCode);
		deleteLast(sb);
		return sb.toString();
	}

}

File: examples/eAPI v1.8/dotNET/CsobGTWDemo/CsobGatewayClientExample/Communication/DataObjects/PayRes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CsobGatewayClientExample.Communication.DataObjects
{
    public class PayRes : SignBase
    {
        public string PayId { get; set; }

        public int ResultCode { get; set; }

        public string ResultMessage { get; set; }

        public int PaymentStatus { get; set; }

        public string AuthCode { get; set; }

        public List<string> Extensions { get; set; }

        public PayRes()
        {
        }

        public override string ToSign()
        {
            var sb = new StringBuilder();
            Add(sb, PayId);
            Add(sb, DateTime);
            Add(sb, ResultCode);
            Add(sb, ResultMessage);
            Add(sb, PaymentStatus);
            Add(sb, AuthCode);
            DeleteLast(sb);
            return sb.ToString();
        }

    }
}
